src/main/java/org/broadinstitute/hellbender/tools/exome/acsconversion/ACSModeledSegmentUtils.java

```diff
@@ -112,7 +112,7 @@ public static ACSModeledSegment convertACNVSegmentToACSSegment(final ACNVModeled
         final int targetCount = genome.getTargets().targetCount(acnvModeledSegment.getInterval());
         final int segLabelCNLOH = 2;
         final int hetCount = genome.getSNPs().targetCount(acnvModeledSegment.getInterval());
-        return new ACSModeledSegment(acnvModeledSegment.getInterval(), ModeledSegment.NO_CALL, targetCount, tau/2.0,
+        return new ACSModeledSegment(acnvModeledSegment.getInterval(), ModeledSegment.NO_CALL, targetCount, ParamUtils.log2(tau/2.0),
```

Comment:
This is somewhat confusing.  I think the local variable should be `segmentMeanInLog2CR = acnvModeledSegment.getSegmentMeanPosteriorSummary().getCenter()` (since this is what the ACSModeledSegment constructor takes), rather than `tau`.  This obviates the need to perform and then immediately undo the transformation locally, which is what you currently do.